Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -435,8 +436,21 @@ DataProcessorSpec specifyExternalFairMQDeviceProxy(char const* name,
       });
     };
     ctx.services().get<CallbackService>().set(CallbackService::Id::Start, channelConfigurationChecker);
+    static int numberOfEoS = 0;
+    numberOfEoS = 0;
```

Comment:
What if we put two proxies into one process? IIRC, this is/will be possible.

Reply:
Ok, I can make the number of EoS received a property of the ChannelInfo. Let me refactor this a  bit.